packages/patternfly-4/react-core/src/components/Modal/Modal.tsx

```diff
@@ -73,7 +73,7 @@ export class Modal extends React.Component<ModalProps, ModalState> {
   toggleSiblingsFromScreenReaders = (hide: boolean) => {
     const bodyChildren = document.body.children;
     for (const child of Array.from(bodyChildren)) {
-      if (child !== this.container) {
+      if (child !== this.state.container) {
```

Comment:
Accident?

Reply:
This is the change that Boaz added to his PR; I think it needs to be written as `this.state.container` because when it's not, the screenreader can't identify the buttons in the modal.

Reply:
Ah, this makes sense given your refactor of `this.container` -> `this.state.container`. It should have always been in `state` since it affects rendering. Thanks!